dashboard/public/weblab/footer.js

```diff
@@ -30,10 +30,14 @@ var applyFooter = function() {
   }
   if (window.encrypted_channel_id) {
     var viewCodeLink = document.getElementById("pagefooter_view_code");
-    viewCodeLink.href =
-      viewCodeLink.href + "/" + window.encrypted_channel_id + "/view";
+    if (!viewCodeLink.includes(window.encryped_channel_id)) {
```

Comment:
Also for future me: this is misspelled 🤦‍♀️

Reply:
oh good catch!